src/main/java/appeng/core/AppEng.java

```diff
@@ -241,4 +254,48 @@ private void serverStarting( final FMLServerStartingEvent evt )
 	{
 		evt.registerServerCommand( new AECommand( evt.getServer() ) );
 	}
+
+	/**
+	 * Checks if AE2 and the API are provided by the same source.
+	 *
+	 * This is either the same jar or directory in case of indev.
+	 */
+	private void checkForApiConflicts()
+	{
+		final ModAPIManager apiManager = ModAPIManager.INSTANCE;
+		if( apiManager.hasAPI( API_ID ) )
+		{
+			final ModContainer aeContainer = FMLCommonHandler.instance().findContainerFor( MOD_ID );
+			ModContainer aeApiContainer = null;
+
+			final Iterable<? extends ModContainer> apiList = apiManager.getAPIList();
+			for( ModContainer container : apiList )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'container' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)